<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassword</class>
 <widget class="QWidget" name="changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldPwd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldCnfrmPwd">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>130</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonChangePassword">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
